integration/e2e_test.go

```diff
@@ -299,3 +302,59 @@ func deployMicrobot(t testing.TB) {
 		}
 	}
 }
+
+func upgradeCluster(t testing.TB, version string) {
+	t.Logf("Upgrading cluster to %s", version)
+	// Patch contol plane machine upgrades.
+	patch_command := fmt.Sprintf("kubectl patch --type=merge microk8scontrolplanes.controlplane.cluster.x-k8s.io microk8s-openstack-control-plane --patch {\"spec\":{\"version\":\"%s\"}}", version)
+	command := strings.Fields(patch_command)
+	cmd := exec.Command(command[0], command[1:]...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Error(string(output))
+		t.Fatalf("Failed to merge the patch to control plane. %s", err)
+	}
+
+	// Patch worker machine upgrades.
+	patch_command = fmt.Sprintf("kubectl patch --type=merge machinedeployments.cluster.x-k8s.io microk8s-openstack-md-0 --patch {\"spec\":{\"template\":{\"spec\":{\"version\":\"%s\"}}}}", version)
+	command = strings.Fields(patch_command)
+	cmd = exec.Command(command[0], command[1:]...)
+	output, err = cmd.CombinedOutput()
+	if err != nil {
+		t.Error(string(output))
+		t.Fatalf("Failed to merge the patch to the machine deployments. %s", err)
+	}
+
+	time.Sleep(30 * time.Second)
+
+	// Now all the machines should be upgraded to the new version.
+	attempt := 0
+	max_attempts := 60
+	command = strings.Fields("kubectl get machine --no-headers")
+	for {
+		cmd := exec.Command(command[0], command[1:]...)
+		output, err := cmd.CombinedOutput()
+		output_str := string(output)
+		if err != nil {
+			t.Log(output_str)
+			if attempt >= max_attempts {
+				t.Fatal(err)
+			} else {
```

Comment:
t.Fatal will terminate the program, so the else block below can be without the else

```
if cond {
   terminate
} else {
   something else
}
```
should instead be
```
if cond {
  terminate
}
something else
```